hermes-2.cxx

```diff
@@ -3384,11 +3388,21 @@ int Hermes::rhs(BoutReal t) {
   if (radial_buffers) {
     /// Radial buffer regions
 
-    // Calculate flux sZ averages
-    Field2D PeDC = averageY(DC(Pe));
-    Field2D PiDC = averageY(DC(Pi));
-    Field2D NeDC = averageY(DC(Ne));
-    Field2D VortDC = averageY(DC(Vort));
+    // Calculate flux Z averages.
+    // This is used for both inner and outer boundaries
+    Field2D PeDC = DC(Pe);
+    Field2D PiDC = DC(Pi);
+    Field2D NeDC = DC(Ne);
+    Field2D VortDC = DC(Vort);
+    Field2D NViDC = DC(NVi);
+
+    // Flux surface averages.
+    // In the core region it can be desirable to damp towards a flux surface average
+    Field2D PeInner = radial_inner_averagey ? averageY(PeDC) : PeDC;
+    Field2D PiInner = radial_inner_averagey ? averageY(PiDC) : PiDC;
+    Field2D NeInner = radial_inner_averagey ? averageY(NeDC) : NeDC;
+    Field2D VortInner = radial_inner_averagey ? averageY(VortDC) : VortDC;
+    Field2D NViInner = radial_inner_averagey ? averageY(NViDC) : NViDC;
```

Comment:
Do you want to have an option to damp `NVi` toward zero on a core boundary, like you had before this PR? Could add another input flag like `zero_inner_NVi` and set `NViInner` to 0.0 if it's true.